docs/groovy-script/mods/mwc/crafting_station.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "MWC Crafting"
+titleTemplate: "Modern Warfare Cubed | CleanroomMC"
+description: "Converts multiple items into another item, with a way to recover some of original resources when dismantled. recipes with group `BULLET, MAGAZINE, GRENADE` can be only crafted via Ammo Press, while recipes with `GUN, ATTACHMENT_NORMAL, ATTACHMENT_MODIFICATION` categories can only be crafted with Crafting Station."
+source_code_link: "https://github.com/Cubed-Development/Modern-Warfare-Cubed/blob/next/src/main/java/com/paneedah/mwc/groovyscript/script/CraftingStation.java"
+---
+
+# MWC Crafting (Modern Warfare Cubed)
+
+## Description
+
+Converts multiple items into another item, with a way to recover some of original resources when dismantled. recipes with group `BULLET, MAGAZINE, GRENADE` can be only crafted via Ammo Press, while recipes with `GUN, ATTACHMENT_NORMAL, ATTACHMENT_MODIFICATION` categories can only be crafted with Crafting Station.
+
+## Identifier
+
+Refer to this via any of the following:
+
+```groovy:no-line-numbers {1}
+mods.mwc.crafting_station/* Used as page default */ // [!code focus]
+mods.mwc.craftingstation
+mods.mwc.craftingStation
+mods.mwc.CraftingStation
+mods.mwc.ammo_press
+mods.mwc.ammopress
+mods.mwc.ammoPress
+mods.mwc.AmmoPress
+```
+
+
+## Adding Recipes
+
+### Recipe Builder
+
+Just like other recipe types, the MWC Crafting also uses a recipe builder.
+
+Don't know what a builder is? Check [the builder info page](../../getting_started/builder.md) out.
+
+:::::::::: details mods.mwc.crafting_station.recipeBuilder() {open id="abstract"}
+- `ItemStackList`. Sets the item outputs of the recipe. Requires exactly 1.
+
+    ```groovy:no-line-numbers
+    output(ItemStack)
+    output(ItemStack...)
+    output(Collection<ItemStack>)
+    ```
+
+- `CraftingGroup`. Category of crafting recipe. Requires not null. (Default `GUN`).
+
+    ```groovy:no-line-numbers
+    setGroup(String)
```

Comment:
want to set the priority on this down to a bit under 3000 to get this ordered in a more logical way.